gradleops/AndroidSeed/tealeaf/src/main/obj/local/armeabi/objs/zip/libzip/zip_stat_index.o.d

```diff
@@ -0,0 +1,80 @@
+/Users/alex/codeworks/YG/reposAndroid/devkitworks/TeaLeafGradle/app/src/main/obj/local/armeabi/objs/zip/libzip/zip_stat_index.o: \
```

Comment:
Do these files need to be checked in ?

Reply:
Please confirm if these files. Files that end with .o.d need to be checked in?
What are these files ?